henry_kim/fundamentals/ruby_puzzles.rb

```diff
@@ -0,0 +1,82 @@
+# Create an array with the following values: 3,5,1,2,7,9,8,13,25,32.
+#  Print the sum of all numbers in the array.
+#  Also have the function return an array that only include numbers that are greater than 10
+# (e.g. when you pass the array above, it should return an array with the values of 13,25,32 - hint: use reject or find_all method)
+
+# array = [3,5,1,2,7,9,8,13,25,32]
+# def puzzle_one array
+#   puts array.reduce(:+)
+#   print array.reject {|x| x < 10}
+# end
+#
+# puzzle_one array
+
+# Create an array with the following values: John, KB, Oliver, Cory, Matthew, Christopher.
+# Shuffle the array and print the name of each person. Have the program also return an array with names that are longer than 5 characters.
+
+# names = ["John", "KB", "Oliver", "Cory", "Matthew", "Christopher"]
+#
+# def puzzle_two array
+#   print array.shuffle
+#   puts array.find_all {|i| i.length > 5}.to_s
```

Comment:
No need to convert to_s. puts add a new line at the end of each array element. print or p will tell you that the type is still an array